src/ai/backend/manager/models/gql.py

```diff
@@ -394,6 +399,15 @@ class Queries(graphene.ObjectType):
         access_key=graphene.String(),  # must be empty for user requests
     )
 
+    # super-admin only
+    shared_vfolder_list = graphene.Field(
```

Comment:
I'd like to rename it as `vfolder_share_list` or more directly, `vfolder_permission_list`.

- "Shared vfolder" -> the target object is vfolder.
- "VFolder share" -> the target object is the permission or the fact that it is shared.

Reply:
I renamed it to `vfolder_permission_list`.